conductor_api/src/conductor/base.rs

```diff
@@ -1419,26 +1487,39 @@ pub mod tests {
         .to_string()
     }
 
+    fn bridge_call_fn_declaration() -> FnDeclaration {
+        FnDeclaration {
+            name: String::from("hello"),
+            inputs: vec![],
+            outputs: vec![dna::fn_declarations::FnParameter {
+                name: String::from("greeting"),
+                parameter_type: String::from("String"),
+            }],
+        }
+    }
+
     fn callee_dna() -> Dna {
         let wat = &callee_wat();
         let mut dna = create_test_dna_with_wat("greeter", Some(wat));
         dna.uuid = String::from("basic_bridge_call");
-        dna.zomes.get_mut("greeter").unwrap().add_fn_declaration(
-            String::from("hello"),
-            vec![],
-            vec![dna::fn_declarations::FnParameter {
-                name: String::from("greeting"),
-                parameter_type: String::from("String"),
-            }],
-        );
-        dna.zomes
-            .get_mut("greeter")
-            .unwrap()
-            .traits
-            .get_mut("hc_public")
-            .unwrap()
-            .functions
-            .push("hello".into());
+        let fn_declaration = bridge_call_fn_declaration();
+
+        {
+            let zome = dna.zomes.get_mut("greeter").unwrap();
+            zome.fn_declarations.push(fn_declaration.clone());
+            zome.traits
+                .get_mut("hc_public")
+                .unwrap()
```

Comment:
The idea I guess is these unwraps are unrecoverable? Small change but would prefer expects since they give a little bit more context

Reply:
yah, but remember this is in a test, so if it fails something really weird is going on.

Reply:
Oh yeah it is, Dismiss this comment @lucksus